tests/run/setcomp.pyx

```diff
@@ -62,3 +59,12 @@ def sorted(it):
     l = list(it)
     l.sort()
     return l
+
+def set_comp_scope():
+    """
+    locals should be evaluated in the outer scope
```

Comment:
This test seems better suited for `run/locals.pyx` than the general set comprehension test suite.